cmd/unset.go

```diff
@@ -38,21 +38,21 @@ This will update the current project configuration file.`,
 		}
 	},
 	Run: func(cmd *cobra.Command, args []string) {
-		commandName := args[0]
-
 		configuration, err := config.LoadOneBuildConfiguration()
 		if err != nil {
 			utils.PrintErr(err)
 			return
 		}
 
-		index := indexOfCommandIfPresent(configuration, commandName)
-		if index == -1 {
-			utils.Println("Command '" + commandName + "' not found")
-			return
+		for _, commandName := range args {
```

Comment:
Current Implementation will print `Not found` message for all missing commands.

how about making a single message with list of commands.
ex. 
```markdown
Following command(s) not found: cmd1, cmd2
```

OR
```markdown
Following command(s) not found:
  - cmd1
  - cmd2
```

Preferably with red-colored output.

WDYT about above options?

Reply:
That makes sense. Both versions are okay, though I am leaning slightly towards the one line version. Do you have a preference for either?

Reply:
One liner LGTM.